test/runtests.jl

```diff
@@ -3,135 +3,135 @@ using Documenter
 using Test
 
 @testset "k*pi" begin
-  @test isapprox(2*pi, twoπ)
-  @test isapprox(4*pi, fourπ)
-  @test isapprox(pi/2, halfπ)
-  @test isapprox(pi/4, quartπ)
+    @test isapprox(2*pi, twoπ)
+    @test isapprox(4*pi, fourπ)
+    @test isapprox(pi/2, halfπ)
+    @test isapprox(pi/4, quartπ)
 end
 
 @testset "k/pi" begin
-  @test isapprox(1/pi, invπ)
-  @test isapprox(2/pi, twoinvπ)
-  @test isapprox(4/pi, fourinvπ)
+    @test isapprox(1/pi, invπ)
+    @test isapprox(2/pi, twoinvπ)
+    @test isapprox(4/pi, fourinvπ)
 end
 
 @testset "1/(k*pi)" begin  
-  @test isapprox(1/(2pi), inv2π)
-  @test isapprox(1/(4pi), inv4π)
+    @test isapprox(1/(2pi), inv2π)
+    @test isapprox(1/(4pi), inv4π)
 end
 
 @testset "sqrt" begin
-  @test isapprox(sqrt(2), sqrt2)
-  @test isapprox(sqrt(3), sqrt3)
-  @test isapprox(sqrt(pi), sqrtπ)
-  @test isapprox(sqrt(2pi), sqrt2π)
-  @test isapprox(sqrt(4pi), sqrt4π)
-  @test isapprox(sqrt(pi/2), sqrthalfπ)
-  @test isapprox(sqrt(1/2), invsqrt2)
-  @test isapprox(sqrt(1/(pi)), invsqrtπ)
-  @test isapprox(sqrt(1/(2pi)), invsqrt2π)
+    @test isapprox(sqrt(2), sqrt2)
+    @test isapprox(sqrt(3), sqrt3)
+    @test isapprox(sqrt(pi), sqrtπ)
+    @test isapprox(sqrt(2pi), sqrt2π)
+    @test isapprox(sqrt(4pi), sqrt4π)
+    @test isapprox(sqrt(pi/2), sqrthalfπ)
+    @test isapprox(sqrt(1/2), invsqrt2)
+    @test isapprox(sqrt(1/(pi)), invsqrtπ)
+    @test isapprox(sqrt(1/(2pi)), invsqrt2π)
 end
 
 @testset "log" begin
-  @test isapprox(log(1/2), loghalf)
-  @test isapprox(log(2), logtwo)
-  @test isapprox(log(10), logten)
-  @test isapprox(log(pi), logπ)
-  @test isapprox(log(2pi), log2π)
-  @test isapprox(log(4pi), log4π)
+    @test isapprox(log(1/2), loghalf)
+    @test isapprox(log(2), logtwo)
+    @test isapprox(log(10), logten)
+    @test isapprox(log(pi), logπ)
+    @test isapprox(log(2pi), log2π)
+    @test isapprox(log(4pi), log4π)
 end
 
 @testset "type system" begin
-  @test twoπ === IrrationalConstants.Twoπ()
-  @test twoπ isa IrrationalConstants.IrrationalConstant
-  @test twoπ isa AbstractIrrational
+    @test twoπ === IrrationalConstants.Twoπ()
+    @test twoπ isa IrrationalConstants.IrrationalConstant
+    @test twoπ isa AbstractIrrational
 end
 
 @testset "hash" begin
-  for i in (twoπ, invπ, sqrt2, logtwo)
-    for j in (twoπ, invπ, sqrt2, logtwo)
-      @test isequal(i==j, hash(i)==hash(j))
+    for i in (twoπ, invπ, sqrt2, logtwo)
+        for j in (twoπ, invπ, sqrt2, logtwo)
+            @test isequal(i==j, hash(i)==hash(j))
+        end
     end
-  end
 end
 
 @testset "doctests" begin
-  DocMeta.setdocmeta!(
+    DocMeta.setdocmeta!(
     IrrationalConstants, :DocTestSetup, :(using IrrationalConstants); recursive=true
```

Comment:
This seems wrong, should be indented more, I assume?

Reply:
```suggestion
        IrrationalConstants, :DocTestSetup, :(using IrrationalConstants); recursive=true
```

Reply:
I agree with that. I just updated the indents with a standard vscode reindent command. Should we introduce JuliaFormatter?

Reply:
Not sure if it is worth it yet, given the size of the project. In any case I'd prefer a separate PR for it.

Reply:
You mean, VScode uses this weird formatting? I think if you don't add a JuliaFormatter.toml file the Julia VScode extension does not perform any (proper) formatting at all, maybe it just falls back to some generic VScode thing.

Reply:
Let's just indent it manually then for now.